src/components/CommonContent.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import LinksList from './LinksList';
+import ButtonsGroup from './ButtonsGroup';
+import DynamicTable from './DynamicTable';
+import VideoGroup from './VideoGroup';
+import InfoGroup from './InfoGroup';
+import Json from './Json';
+import ContentItemWrapper from './ContentItemWrapper';
+
+const components = {
+  buttons: ButtonsGroup,
```

Comment:
These are the available components that can be defined by `type` in content files